server/properties.js

```diff
@@ -668,4 +668,10 @@ router.get ('/:urn/ids/*', function (req, res) {
 		}) ;
 }) ;
 
-module.exports =router ;
+router.post('/refreshToken', (req, res) => {
```

Comment:
@nick-ooms-neanex You can destructur the `req` variable right in the function arguments

```suggestion
router.post('/refreshToken', ({ body: { clientId, urn, token } }, res) => {
```